oci/namespaces.go

```diff
@@ -4,13 +4,16 @@ import specs "github.com/opencontainers/runtime-spec/specs-go"
 
 // RemoveNamespace removes the `nsType` namespace from OCI spec `s`
 func RemoveNamespace(s *specs.Spec, nsType specs.NamespaceType) {
-	idx := -1
```

Comment:
It works and looks very concise.
But what's wrong with my current code? I think it's easier to understand